packages/office-ui-fabric-react/src/components/Breadcrumb/Breadcrumb.tsx

```diff
@@ -66,16 +66,16 @@ export class Breadcrumb extends BaseComponent<IBreadcrumbProps, IBreadcrumbState
   }
 
   public render() {
-    let { className } = this.props;
+    let { className, navigationLandmarkAriaLabel } = this.props;
```

Comment:
Actually just going to move both role="navigation" and the aria-label onto the div that surrounds both the focus zone and the overflow buttons